### PR TITLE
Erik screenings component on startpage

### PR DESCRIPTION
**Pull request by NelsonTheBrave**
This is my component I've been working on for a while. It works OK but still has some issues, the major one being that there it tries to overwrite the screenings model if you reload the page, which means you need to restart the dev server when you want to fetch again. This is needs to be fixed at a later point, but want to share my work so others can check the code. I think the PR could be approved even with this error that can be fixed in another branch.

It also still needs to have links to the bookings and pages of individual movies, which will also be added later.

image

### CODE REVIEW

**Comment by FelixHultman** [comment]
It looks good from what i can see, apart from the things you mention yourself and will be fixed at a later stage. There seem to be some formatting problems but hopefully that should already be fixed after our last meeting. The checks that fails seems to be the qoute problem we looked at on the "om oss" page, should be easy to fix maybe ?

File: app/page.tsx [hunk 1]
```diff
@@ -1,6 +1,10 @@
+'use client'
+import DatePickerStartpage from "./ui/datePickerStartpage";
```
Comment (FelixHultman) [inline app/page.tsx]: This import does not seem to exist, or am i missing something ?

**Comment by JrSchoolAccount** [comment]
./app/page.tsx:2:33
Type error: Cannot find module './ui/datePickerStartpage' or its corresponding type declarations.
``` js
  1 | 'use client'
> 2 | import DatePickerStartpage from "./ui/datePickerStartpage";
    |                                 ^
  3 | import ScreeningsTableStartpage from "./ui/screeningsTableStartpage";
  4 |
  5 | export default function Home() {
Error: Process completed with exit code 1.
```

Finns lite att fixa 😃

**Review by FelixHultman — APPROVED** [review]

**Review by rl-86 — APPROVED** [review]
I have tested the branch manually in browser and it fetches screenings correctly and selecting dates works as intended.
However, the database connection crashes( I think ) when I´m going back to the front page after visiting a page that doesn't exist and returns 404. 

Error:
Skärmbild 2024-05-13 131005

But I believe you are already working on a solution for this issue or know how to fix it.

I like the design and layout, it just need responsiveness for smaller screens.

Otherwise, great work! 👍

File: app/page.tsx (hunk 1 above)
Comment (NelsonTheBrave) [reply app/page.tsx]: No, Jesper removed it and committed afterwards. I'm not sure how you found it? Are you looking at an older commit?

**Comment by NelsonTheBrave** [comment]
Thanks for the feedback, I got help to fix the issue with not being able to reload browser without crashing, so now it should work well. I'll merge

**Pull request merged**